src/Sprite/TilesetLoader.cpp

```diff
@@ -117,7 +117,7 @@ namespace Tileset
 
 	void ValidateTileset(const BitmapFile& tileset)
 	{
-		constexpr uint32_t DefaultPixelWidth = 32;
+		constexpr int32_t DefaultPixelWidth = 32;
 		constexpr uint32_t DefaultPixelHeightMultiple = DefaultPixelWidth;
```

Comment:
Curiously the width and height values are now of different type. Not sure that will lead to any problems, just that it's unexpected.